sdk/personalizer/Azure.AI.Personalizer/src/Models/PersonalizerClient.cs

```diff
@@ -70,13 +72,25 @@ public PersonalizerClient(Uri endpoint, TokenCredential credential, Personalizer
         /// <param name="endpoint"> Supported Cognitive Services endpoint. </param>
         /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
         /// <param name="isLocalInference"> A flag to determine whether to use local inference. </param>
+        /// <param name="interactionsSubsamplePercentage"> Percentage from (0,1] determines how much percentage of interaction events to consider </param>
+        /// <param name="observationsSubSamplePercentage"> Percentage from (0,1] determines how much percentage of observation events to consider </param>
         /// <param name="options"> The options for configuring the client. </param>
-        public PersonalizerClient(Uri endpoint, TokenCredential credential, bool isLocalInference, PersonalizerClientOptions options = null) :
+        public PersonalizerClient(Uri endpoint, TokenCredential credential, bool isLocalInference, float interactionsSubsamplePercentage = 100, float observationsSubSamplePercentage = 100, PersonalizerClientOptions options = null) :
```

Comment:
default value should be 1.0f. Also rename to ***Rate.

Reply:
corrected it